ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixClusterManager.java

```diff
@@ -128,6 +130,7 @@ public void run() {
       initializationException.compareAndSet(null, e);
     }
     if (initializationException.get() == null) {
+      sealedStateChangeCounter.incrementAndGet();
```

Comment:
I think it is unnecessary, but if you guys find that more intuitive, will make the change.